app/[sheetId]/List.tsx

```diff
@@ -105,7 +105,7 @@ const LinkBlock = ({
         return (
           <div key={it.title} className="link text-action text-sm">
             <a target="_blank" href={myMatch?.[2] ?? '#'}>
-              {myMatch?.[1]}{' '}
+              {myMatch?.[1] || 'Open Link'}{' '}
```

Comment:
I don't think this will work because this is expecting that you have [something](something else)

in this case you will see Open link (this should probably come from meta btw) but the link will be broken. `myMatch?.[2]` will not exist either so  the link will always point to `'#'`